data/src/main/java/pl/edu/agh/model/UserRole.java

```diff
@@ -0,0 +1,5 @@
+package pl.edu.agh.model;
+
+public enum UserRole {
+    ADMIN
```

Comment:
O, i przecinki po ostatniej wartości enuma też ktoś we mnie wytrenował kiedyś. W sumie nie wiem czemu - [internet](https://kariem.github.io/2016/05/04/trailing-comma-java/) mówi, że diff jest potem czytelniejszy i w sumie nie kłamie

Reply:
@jplewa to jest chyba subiektywne bardzo, ja bym się nie czepiał

Reply:
No pewnie, że subiektywne, ale jest w tym jakiś sens

![obraz](https://user-images.githubusercontent.com/34098778/55513096-36a55000-5665-11e9-8ffa-8949c8e41171.png)
![obraz](https://user-images.githubusercontent.com/34098778/55513103-39a04080-5665-11e9-950d-7680b255eb81.png)

[źródło](https://medium.com/@nikgraf/why-you-should-enforce-dangling-commas-for-multiline-statements-d034c98e36f8)

Reply:
a może planuję dodawać wartości w jednej linii? ;)